README.md

```diff
@@ -432,6 +504,48 @@ The SDK provides `IKUpload` component to upload files to the ImageKit Media Libr
 Sample Usage
 
 ```js
+const reftest = useRef(null);
```

Comment:
There is not much difference between this example and the one below. We should remove custom button implementation from one the first example to keep it simple.